src/functions/Functions.php

```diff
@@ -93,13 +91,6 @@ public function countArguments(): array
      */
     public function countArgumentsWrapper(): array
     {
-        $arg = func_get_args();
-        foreach ($arg as $value) {
-            if (!is_string($value)) {
-                throw new \InvalidArgumentException('String are allowed only, bro!');
```

Comment:
why do you delete this code ?

Reply:
I fetched upstream where this code was deleted and didn't noticed this. Fixed.